src/pages/index.js

```diff
@@ -65,35 +86,128 @@ profileEditButton.addEventListener("click", () => {
   editPopupForm.open();
 });
 
+//Avatar Profile change
+avatarEditButton.addEventListener("click", () => {
+  avatarFormValidator.toggleButtonState();
+  avatarProfilePopupForm.open();
+});
+
 /*Functions*/
 /*Cards*/
 function renderCard(cardData) {
-  const card = new Card(cardData, "#card-template", handleCardImageClick);
-  section.addItem(card.getCardElement());
+  const card = new Card({
+    data: cardData,
+    templateSelector: "#card-template",
+    handleCardImageClick,
+    myUserId: userId,
+    handleDeleteCard,
+    handleLikeButton: (cardId) => {
+      if (card.isLiked()) {
+        api
+          .removeLike(cardId)
+          .then((data) => {
+            card.updateLikes(data.likes);
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      } else {
+        api
+          .addLike(cardId)
+          .then((data) => {
+            card.updateLikes(data.likes);
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      }
+    },
+  });
+  return card.getCardElement();
+}
+let cardList;
+api
+  .getInitialCards()
```

Comment:
You need to combine the requests of the user's info and the cards together in `Promise.all` because it might happen so that the `_id` has not been received yet but the cards have and your cards will be rendered not correctly. 
More info is here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

```js

Promise.all([getUserInfo(), getCards()])
// destructure the response
  .then(([userData, cards]) => {
      // set all the data
  })
  .catch(err => {
    // catch possible errors
  });

```